src/webhint-theme/source/images/openjsf.svg

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you use some tools like: https://jakearchibald.github.io/svgomg/ to reduce the size of the logo?

Reply:
I mean the size in bytes not in px hehehe.